### DISCUSSION TITLE
'View Only' permissions for a given User ... how?

### DISCUSSION THREAD

**Discussion by Ltek** [fluidd-core/fluidd#1266] [Q&A]
I read the Wiki and did some web searches -- dont see how to do this?  I want to grant my friend and my kids a way to view but not mess anything up. thx!

**Comment by pedrolamas** [comment]
Fluidd is just a web interface, it does whatever Moonraker (the API server) allows it to do. Having said that, right now there is no such option on Moonraker... you can enable authentication, but once authenticated all users have full control.

There is however an alternative that I can think of: if you use Home Assistant, you can create a custom Dashboard with whatever controls you want for your printer!

**Reply by Ltek** [reply]
Thank you for the quick reply. Since I dont use HA I'll look into how I can provide a web stream URL proxy. 
And, I'll put in a feature request with Moonraker repo, I'm pretty surprised it doesnt have this. I suspect a lot of people might use it.

thx again!

**Reply by Ltek** [reply]
I narrowed down the ask... https://github.com/Arksine/moonraker/issues/764
